xwiki-pro-macros-ui/src/main/resources/Confluence/Macros/Expand.xml

```diff
@@ -0,0 +1,566 @@
+      <id>
+        <disabled>0</disabled>
+        <name>id</name>
+        <number>1</number>
+        <prettyName>Macro id</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </id>
+      <name>
+        <disabled>0</disabled>
+        <name>name</name>
+        <number>2</number>
+        <prettyName>Macro name</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </name>
+      <priority>
+        <disabled>0</disabled>
+        <name>priority</name>
+        <number>11</number>
+        <numberType>integer</numberType>
+        <prettyName>Priority</prettyName>
+        <size>10</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.NumberClass</classType>
+      </priority>
+      <supportsInlineMode>
+        <disabled>0</disabled>
+        <displayFormType>select</displayFormType>
+        <displayType>yesno</displayType>
+        <name>supportsInlineMode</name>
+        <number>5</number>
+        <prettyName>Supports inline mode</prettyName>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </supportsInlineMode>
+      <visibility>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <freeText>forbidden</freeText>
+        <largeStorage>0</largeStorage>
+        <multiSelect>0</multiSelect>
+        <name>visibility</name>
+        <number>6</number>
+        <prettyName>Macro visibility</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator>|</separator>
+        <separators>|</separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>Current User|Current Wiki|Global</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </visibility>
+    </class>
+    <property>
+      <async_cached>0</async_cached>
+    </property>
+    <property>
+      <async_context/>
+    </property>
+    <property>
+      <async_enabled>0</async_enabled>
+    </property>
+    <property>
+      <code>{{velocity}}
+$xwiki.ssx.use('Confluence.Macros.Expand')
+#if (!$expandMacroColllapseId)
+  #set ($expandMacroColllapseId = 0)
+#else
+  #set ($expandMacroColllapseId = $expandMacroColllapseId + 1)
+#end
+#set ($opened = $xcontext.action == 'edit')
+#set ($accorionId = "accordion-$expandMacroColllapseId")
+#set ($toggleId = "toggle-$expandMacroColllapseId")
+#set ($expandId = "collapse-$expandMacroColllapseId")
+
+{{html clean="false"}}
+&lt;div class="panel-group confluence-expand-macro" id="${accorionId}" role="tablist"&gt;
+  &lt;div class="panel panel-default"&gt;
+    &lt;div class="panel-heading" role="tab" id="${toggleId}"&gt;
+      &lt;h4 class="panel-title"&gt;
```

Comment:
Not sure if it's absolutely needed, but as this is the title of the expand, I should be a good idea to indicate it semantically. The only issue I see with this is that the heading level is not going to be coherent with the rest of the page. In bootstrap, for that kind of titles (like modal titles), `h4` is used (and I saw this approach in other design systems too). So that's why I went for a `h4` for the title of this expand.